### ISSUE TITLE
How to generate a single pic?

### ISSUE THREAD

**Issue by sanersbug** [POSTECH-CVLab/PyTorch-StudioGAN#105]
Thanks for your contribution!  when i trained the model , I use the follow order to generate result.
` CUDA_VISIBLE_DEVICES=0,...,N python3 src/main.py -iv -std_stat --standing_step 1233 -c src/configs/M_dataset.json --checkpoint_folder checkpoints/xxx/`

the result is a 1*8 imgs canvas , it contains 8 results ,but how to generate only one image?
and when i generate the result , why the batchsize must be devided by 8 in the worker.py about the run_image_visualization?

**Comment by MINGUKKANG** [comment]
the result is a 1*8 imgs canvas , it contains 8 results ,but how to generate only one image?
=> please use -s option instead.

when i generate the result , why the batchsize must be devided by 8 in the worker.py about the run_image_visualization?
=> for plotting an image canvas which consists of n \times 8 images.

Thank you!

**Comment by sanersbug** [comment]
@MINGUKKANG Thanks for your reply!

**Comment by asif-nuaa** [comment]
> the result is a 1*8 imgs canvas , it contains 8 results ,but how to generate only one image? => please use -s option instead.

I also want to generate single image, but -s option is no more available to date. Current usage of command is given as 

usage: main.py [-h] [--entity ENTITY] [--project PROJECT] [-cfg CFG_FILE] [-data DATA_DIR]
               [-save SAVE_DIR] [-ckpt CKPT_DIR] [-best] [--seed SEED] [-DDP] [--backend BACKEND]
               [-tn TOTAL_NODES] [-cn CURRENT_NODE] [--num_workers NUM_WORKERS] [-sync_bn] [-mpc]
               [--truncation_factor TRUNCATION_FACTOR] [--truncation_cutoff TRUNCATION_CUTOFF]
               [-batch_stat] [-std_stat] [-std_max STANDING_MAX_BATCH] [-std_step STANDING_STEP]
               [--freezeD FREEZED] [-lgv] [-lgv_rate LANGEVIN_RATE] [-lgv_std LANGEVIN_NOISE_STD]
               [-lgv_decay LANGEVIN_DECAY] [-lgv_decay_steps LANGEVIN_DECAY_STEPS]
               [-lgv_steps LANGEVIN_STEPS] [-t] [-hdf5] [-l]
               [-metrics EVAL_METRICS [EVAL_METRICS ...]] [--pre_resizer PRE_RESIZER]
               [--post_resizer POST_RESIZER] [--num_eval NUM_EVAL] [-sr] [-sf]
               [-sf_num SAVE_FAKE_IMAGES_NUM] [-v] [-knn] [-itp] [-fa] [-tsne] [-ifid]
               [--GAN_train] [--GAN_test] [-resume_ct] [-sefa] [-sefa_axis NUM_SEMANTIC_AXIS]
               [-sefa_max MAXIMUM_VARIATIONS] [-empty_cache] [--print_freq PRINT_FREQ]
               [--save_freq SAVE_FREQ] [--eval_backbone EVAL_BACKBONE] [-ref REF_DATASET]
               [--calc_is_ref_dataset]
